frontend/src/metabase/visualizations/components/settings/ChartSettingInputNumeric.tsx

```diff
@@ -13,10 +14,12 @@ const ChartSettingInputNumeric = ({
 }: ChartSettingInputProps) => {
   const [internalValue, setInternalValue] = useState(value);
 
+  const { onChangeSettings, ...inputProps } = props;
```

Comment:
Fixing a console error about onChangeSettings not being recognized on input elements